6.2.2000run.py

```diff
@@ -0,0 +1,15 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    """
+    :param f: function.
+    :param args: args to pass to the function.
+    :param kwargs: kwargs to pass to the function.
+    :return: The amount of time passed between starting the function received and ending it.
+    """
+    start_time = time.time()
```

Comment:
time is not an accurate enough method in order to measure the speed of a simple pythonic function.
Read the time module documentation in order to find a better method for timing a function.